core/src/CommonRestartMetadata.cpp

```diff
@@ -15,24 +15,23 @@
 
 namespace Nextsim {
 
-netCDF::NcGroup& CommonRestartMetadata::writeStructureType(
-    netCDF::NcFile& rootGroup, const ModelMetadata& metadata)
+// TODO: Don't pass ncfile in and out
+netCDF::NcFile& CommonRestartMetadata::writeStructureType(
+    netCDF::NcFile& ncfile, const ModelMetadata& metadata)
 {
-    netCDF::NcGroup structGroup = rootGroup.addGroup(IStructure::structureNodeName());
-    structGroup.putAtt(IStructure::typeNodeName(), metadata.structureName());
-    return rootGroup;
+    ncfile.putAtt(IStructure::typeNodeName(), metadata.structureName());
+    return ncfile;
 }
 
-netCDF::NcGroup& CommonRestartMetadata::writeRestartMetadata(
-    netCDF::NcGroup& metaGroup, const ModelMetadata& metadata)
+// TODO: Don't pass ncfile in and out
+netCDF::NcFile& CommonRestartMetadata::writeRestartMetadata(
+    netCDF::NcFile& ncfile, const ModelMetadata& metadata)
```

Comment:
Note to self - this still needs doing.

Reply:
We discussed this in the meeting today and decided it's not necessarily an issue because the signature already does a similar thing.